### PR TITLE
Add awaitReady in Modal file

### PR DESCRIPTION
**Pull request by aweng98**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by aweng98** [comment]
jenkins retest

**Review by breilly2 — APPROVED** [review]
:+1: Didn't test but seems reasonable to me.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala [hunk 1]
```diff
@@ -6,10 +6,12 @@ import org.broadinstitute.dsde.firecloud.component._
 import org.broadinstitute.dsde.firecloud.page.MethodTable
 import org.openqa.selenium.WebDriver
 
-class ImportMethodConfigModal(implicit webDriver: WebDriver) extends Modal("import-method-configuration-modal") {
+class ImportMethodConfigModal(implicit webDriver: WebDriver) extends OKCancelModal("import-method-configuration-modal") {
```
Comment (MattPutnam) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala]: OKCancelModal assumes the existence of OK/cancel buttons. Actually, given the way this testing works, there's no real need for the `Modal` vs `OKCancelModal` distinction, so we should probably fold them together and just call it `Modal`.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala [hunk 2]
```diff
@@ -6,10 +6,12 @@ import org.broadinstitute.dsde.firecloud.component._
...
   private val chooseConfigFromRepoButton = Button("import-from-repo-button" inside this)
   private val copyConfigFromWorkspaceButton = Button("copy-from-workspace-button" inside this)
 
+  override def awaitReady(): Unit = chooseConfigFromRepoButton.awaitVisible()
```
Comment (MattPutnam) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala]: This shouldn't be necessary. This modal doesn't do any loading and should be ready as soon as it renders. If this change is actually helping something, we have serious problems with the entire framework.

Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala]: I believe I saw a test failed with a screenshot of this Modal last week, and it led me to this Modal file. However, I don't have that screenshot saved to show you.

Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala]: I'm fine with not adding this check now but I can keep an eye on MethodConfigSpec test.

**Comment by aweng98** [comment]
jenkins retest

**Review by andy7i — APPROVED** [review]
👍 assuming tests pass

**Comment by aweng98** [comment]
jenkins retest

**Pull request merged**